### PR TITLE
Added node scripts for handling with data generation

### LINKED ISSUE #22: Create CLI script for generator
Currently we have a single NPM script for data generation purposes. However no script files exists. We would like to create a script files that replace the data generation process.

**Tasks:**

 - [x] Create a script file at: `src/server/scripts/generate-data.js`.
 - [x] Follow [this guide](https://medium.com/netscape/a-guide-to-create-a-nodejs-command-line-package-c2166ad0452e) to add the proper shebang directive and setup permissions only. Do not add a `bin` entry.
 - [x] The script file should support a `--only` option and only one of two values are accepted: `users` or `repos`. When that CLI option is specified, then only the specified set will be generated.
 - [x] Replace the current single NPM script (`generate`, `scrape:users` and `scrape:repos`) with one script: `generate-data` which will execute our new cli script defined at: `src/server/scripts/generate-data.js`.

### PR DESCRIPTION
**Pull request by jouderianjr**
Resolves #22

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,9 +35,9 @@
   "scripts": {
     "sequelize": "sequelize",
     "scrape": "node -e \"require('./src/server/services/scraper').scrape()\"",
-    "scrape:users": "node -e \"require('./src/server/services/scraper/users').scrape()\"",
-    "scrape:repos": "node -e \"require('./src/server/services/scraper/repos').scrape()\"",
-    "generate": "node -e \"require('./src/server/services/generator').generate()\"",
+    "scrape:users": "./src/server/services/scripts/generate-data.js --only users",
+    "scrape:repos": "./src/server/services/scripts/generate-data.js --only repos",
```
Comment (rmariuzzo) [inline package.json]: @jouderianjr the scraper scripts are for another stuff, we don't want to replace it with _data generation_. Please, restore `scrape:users` and `scrape:repos`.

File: src/server/services/scripts/generate-data.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+if (args.length === 0) {
+  require('./../generator').generate()
+} else if (args.length === 2) {
+
+  if ( args[0] === '--only') {
+
+    switch (args[1]) {
+      case 'users':
+        require('./../scraper/users').scrape()
```
Comment (rmariuzzo) [inline src/server/services/scripts/generate-data.js]: The generator file has three methods: `generate`, `generateUsers` and `generateRepos`. We don't want to include operations from the scraper.

Please, require the `./../generator` file before the `args` declaration and use the aforementioned methods.

**Comment by jouderianjr** [comment]
Oh, got it, I replaced the scrape scripts because you said in the issue :D #22 but no problem mate, I'll fix it.

Should I create `generate:users` and `generate:repos` commands? 😄

**Comment by rmariuzzo** [comment]
Hey @jouderianjr! Not needed, with just `generate` NPM script will be enough. The others will be called using the same as:

```sh
yarn generate-data
yarn generate-data -- --only=users
yarn generate-data -- --only=repos
```

Please, double check the name for NPM script will be `generate-data` instead. We want to be a little bit more specific.

**Comment by jouderianjr** [comment]
@rmariuzzo I just updated, but I have a question, do you think use: `yarn generate-data --only=repos` is better than `yarn generate-data --only repos?

Ps. Using yarn, the `--` before options is not needed anymore

**Comment by rmariuzzo** [comment]
@jouderianjr if that's not needed anymore then let's celebrate for the shorter and cleaner version!

**Comment by rmariuzzo** [comment]
Thank you @jouderianjr!

**Comment by jouderianjr** [comment]
Thanks mate 💃

**Pull request merged**